### ISSUE TITLE
Add sent links group to right sidebar (in addition to media etc.)

### ISSUE THREAD

**Issue by e-caste** [nextcloud/spreed#7660]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

**Is your feature request related to a problem? Please describe.**
It is simply a slight UX enhancement request.

**Describe the solution you'd like**
The sidebar on the right hand side of the conversation has been greatly enhanced with the latest Talk update, which has added the Media, Files, Voice messages, Audio, and Locations groups to quickly reach those items in the conversation.  
It would be great if there was also another group, Links, to get the list of all the web links / urls that were sent in the conversation.

**Describe alternatives you've considered**
Not relevant.

**Additional context**
Many other chat apps do it and I find it really useful.

### Ecosystem
- [ ] iOS https://github.com/nextcloud/talk-ios/issues/847
- [ ] Android maybe https://github.com/nextcloud/talk-android/issues/2029

**Comment by nickvergessen** [comment]
Yeah, the problem is links are not unique messages like the file shares are. So a message can have multiple links and the current setup of the tabs doesn't work that way.

**Comment by jancborchardt** [comment]
Examples of how it is done in other apps:
- WhatsApp: https://faq.whatsapp.com/842274731791640/?cms_platform=web
- Teams: https://office365itpros.com/2024/06/28/shared-tab-teams-chat/
- Telegram: https://telegram.org/blog/shared-links
